CONTRIBUTING.md

```diff
@@ -296,6 +299,78 @@ follow the Carbon documentation and coding styles.
     -   Keep in mind that code contribution guidelines are incomplete while we
         start work on Carbon, and may change later.
 
+#### How to say things
+
+-   Treat others with respect
+-   Recognize that other points of view are valid, and most decisions are about
+    choosing the best set of trade-offs or closest alignment with Carbon's goals
+    since there usually isn't a single best answer. It is important to frame
+    feedback and discussion about someone else's proposal under the assumption
+    that they too have deep experience in the area but may have come to a
+    different conclusion.
+-   Be clear when something is an opinion by using "I" or "me":
+    -   Not as helpful: `foo` is objectively better word to use
+    -   More helpful: I find `bar` confusing since it has this alternate
+        meaning, I think `foo` is clearer.
```

Comment:
```suggestion
    -   Not as helpful: "`foo` is objectively better word to use"
    -   More helpful: "I find `bar` confusing since it has this alternate
        meaning, I think `foo` is clearer."
```